### PR TITLE
Updated supported Python versions.

### PR DESCRIPTION
**Pull request by felixxm**

### CODE REVIEW

**Comment by felixxm** [comment]
I hope this will fix "Run benchmarks" error:
```
Beginning benchmarks...
Traceback (most recent call last):
  File "/home/runner/.local/bin/asv", line 8, in <module>
    sys.exit(main())
  File "/home/runner/.local/lib/python3.10/site-packages/asv/main.py", line 38, in main
    result = args.func(args)
  File "/home/runner/.local/lib/python3.10/site-packages/asv/commands/__init__.py", line 49, in run_from_args
    return cls.run_from_conf_args(conf, args)
  File "/home/runner/.local/lib/python3.10/site-packages/asv/commands/continuous.py", line 75, in run_from_conf_args
    return cls.run(
  File "/home/runner/.local/lib/python3.10/site-packages/asv/commands/continuous.py", line 114, in run
    result = Run.run(
  File "/home/runner/.local/lib/python3.10/site-packages/asv/commands/run.py", line 294, in run
    Setup.perform_setup(environments, parallel=parallel)
  File "/home/runner/.local/lib/python3.10/site-packages/asv/commands/setup.py", line 89, in perform_setup
    list(map(_create, environments))
  File "/home/runner/.local/lib/python3.10/site-packages/asv/commands/setup.py", line 21, in _create
    env.create()
  File "/home/runner/.local/lib/python3.10/site-packages/asv/environment.py", line 704, in create
    self._setup()
  File "/home/runner/.local/lib/python3.10/site-packages/asv/plugins/conda.py", line 174, in _setup
    self._run_conda(['env', 'create', '-f', env_file_name,
  File "/home/runner/.local/lib/python3.10/site-packages/asv/plugins/conda.py", line 227, in _run_conda
    return util.check_output([conda] + args, env=env)
  File "/home/runner/.local/lib/python3.10/site-packages/asv/util.py", line 754, in check_output
    raise ProcessError(args, retcode, stdout, stderr)
asv.util.ProcessError: Command '/usr/bin/conda env create -f /tmp/tmp9k_zcq7z.yml -p /home/runner/work/django/django/env/06715dc6ea55f2b02b6ebc21bca7e34a --force' returned non-zero exit status 1
```
see [logs](https://github.com/django/django/actions/runs/4816982677/jobs/8577157991?pr=16805).

File: asv.conf.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
-    "version": 1,
+    "version": 2,
```
Comment (smithdc1) [inline asv.conf.json]: I think this should stay as `1`. 

See [logs](https://github.com/django/django-asv/pull/72/checks#step:6:10).

Comment (felixxm) [reply asv.conf.json]: Reverted :+1:

**Comment by smithdc1** [comment]
This still doesn't work. I feel like something changed in the Ubuntu image.

https://github.com/actions/runner-images/pull/7497

This release has a similar date as when the daily workflow started to fail. 

Is conda no longer available? I couldn't see anything like that in the release notes though. 🤷‍♂️

Will be mid next week before I can get any more computer time.

**Comment by smithdc1** [comment]
Fixed in #73

**Pull request closed without merging**